src/test/java/seedu/address/testutil/TypicalIndexes.java

```diff
@@ -9,4 +9,7 @@
     public static final Index INDEX_FIRST_STUDENT = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_STUDENT = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_STUDENT = Index.fromOneBased(3);
+
+    public static final Index INDEX_FIRST_MODULE_CLASS = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_MODULE_CLASS = Index.fromOneBased(2);
```

Comment:
There's quite a bit of overlap here, would be good to deduplicate into:
```suggestion
    public static final Index INDEX_FIRST_ITEM= Index.fromOneBased(1);
    public static final Index INDEX_SECOND_ITEM = Index.fromOneBased(2);
    public static final Index INDEX_THIRD_ITEM = Index.fromOneBased(3);
```